src/command_modules/azure-cli-servicefabric/setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup
+
+VERSION = '2.0.7+dev'
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure-cli-core',
+    'pyOpenSSL'
+]
+
+with open('README.rst', 'r', encoding='utf-8') as f:
+    README = f.read()
+with open('HISTORY.rst', 'r', encoding='utf-8') as f:
+    HISTORY = f.read()
+
+setup(
+    name='azure-cli-servicefabric',
+    version=VERSION,
+    description='Microsoft Azure Command-Line Tools ServiceFabric Command Module',
+    long_description=README + '\n\n' + HISTORY,
+    license='MIT',
+    author='Microsoft Corporation',
+    author_email='azpycli@microsoft.com',
+    url='https://github.com/Azure/azure-cli',
+    classifiers=CLASSIFIERS,
+    namespace_packages=[
+        'azure',
+        'azure.cli',
+        'azure.cli.command_modules'
+    ],
+    packages=[
+        'azure.cli.command_modules.servicefabric'
+    ],
+    install_requires=DEPENDENCIES,
+)
```

Comment:
See https://github.com/Azure/azure-cli/blob/master/doc/authoring_command_modules/example_module_template/setup.py on how this should be structured.